README.md

```diff
@@ -47,6 +47,8 @@ This sets the access token depending upon the environment, you can create one fo
 
 >"CAUTIONARY_ALERT_ENDPOINT": ${cautionaryAlertEndpoint}
 
+>"FEATURE_TOGGLE_ENDPOINT" : ${configurationApiEndpoint}
+
```

Comment:
I'd argue the list of these .env vars within readme is no longer necessary past the creation of the `.env.example` file. It could instead say: "copy the example file, and populate the missing variables."